test/unit/string.spec.ts

```diff
@@ -349,6 +349,20 @@ test.each([`"foobar".length`, `"foobar".repeat(2)`, "`foo${'bar'}`.length", "`fo
     }
 );
 
+test.each(["foo"])("scoped string-union inference (%p)", inp => {
```

Comment:
`.each` with a single case can be removed completely and be inlined to code.

Reply:
Done :)